File: .trae/skills/geo-content/scripts/generate_image.js
#!/usr/bin/env node
/** Compatibility wrapper for GEO content image generation.
 * Canonical implementation: ../geo-content-production/scripts/generate_image.js
 */
const path = require('path');
const cp = require('child_process');
const target = path.resolve(__dirname, '../../geo-content-production/scripts/generate_image.js');
const result = cp.spawnSync(process.execPath, [target, ...process.argv.slice(2)], { stdio: 'inherit' });
if (result.error) {
  console.error(result.error.message || result.error);
  process.exit(1);
}
process.exit(result.status ?? 0);

File: .trae/skills/geo-content/scripts/generate_image.py
#!/usr/bin/env python3
"""Compatibility wrapper for the relocated GEO image generator.

The canonical script lives at:
  ../geo-content-production/scripts/generate_image.py
"""
from pathlib import Path
import runpy

target = Path(__file__).resolve().parents[2] / "geo-content-production" / "scripts" / "generate_image.py"
runpy.run_path(str(target), run_name="__main__")
